db/migrate/20151016151834_add_root_folder_to_courses.rb

```diff
@@ -0,0 +1,36 @@
+class AddRootFolderToCourses < ActiveRecord::Migration
+  # Create root folders for existing courses
+  def up
```

Comment:
Assignment Branch Condition size for up is too high. [22.2/15]<br>Method has too many lines. [24/10]